build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+    id("com.exactpro.th2.gradle.publish") version "0.0.4"
+}
+
+group = "com.exactpro.th2"
+version = project.properties["release_version"].toString()
+
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor(0, "seconds")
+    resolutionStrategy.cacheDynamicVersionsFor(0, "seconds")
+}
+
+dependencies {
+    implementation("com.exactpro.th2:common:5.10.0-dev")
+    implementation("com.exactpro.th2:grpc-common:4.4.0-dev")
```

Comment:
note: There is a newer version 4.5.0-dev with updated gRPC dependenecies. This should resolve issue with OWASP scanning